src/server/server_family.cc

```diff
@@ -89,6 +89,9 @@ ABSL_FLAG(ReplicaOfFlag, replicaof, ReplicaOfFlag{},
           "Specifies a host and port which point to a target master "
           "to replicate. "
           "Format should be <IPv4>:<PORT> or host:<PORT> or [<IPv6>]:<PORT>");
+ABSL_FLAG(uint32_t, slowlog_log_slower_than, 10000,
+          "add commands slower than this value to slowlog");
```

Comment:
`Add commands slower than this threshold to slowlog. The value is expressed in microseconds and if it's negative - disables the slowlog.`
also make it int32_t